department-of-veterans-affairs/caseflow
# Add RampRefiling and the logic to create them

Some more plumbing around the search page. The search page now works for both types of forms.

Also add the initial schema for reentries.

## Testing instructions
- Complete a RAMP Election (demo number, 15555555)
- start a RAMP refiling with the same number
- See that a `RampRefilingIntake` was created, along with a `RampRefiling` connected to the `RampElection`.
- For now it will take you to the RAMP Election review page. This will be fixed in a future PR.
- Attempt to search for a RAMP refiling without a completed RAMP Election (25555555), you should see an error (copy to come in a future ticket)

connects #3965
